datasets/fashion_mnist/README.md

```diff
@@ -0,0 +1,186 @@
+
+## Table of Contents
+
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** [GitHub](https://github.com/zalandoresearch/fashion-mnist)
+- **Repository:** [GitHub](https://github.com/zalandoresearch/fashion-mnist)
+- **Paper:** [arXiv](https://arxiv.org/pdf/1708.07747.pdf)
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+Fashion-MNIST is a dataset of Zalando's article images—consisting of a training set of 60,000 examples and a test set of 10,000 examples. Each example is a 28x28 grayscale image, associated with a label from 10 classes. We intend Fashion-MNIST to serve as a direct drop-in replacement for the original MNIST dataset for benchmarking machine learning algorithms. It shares the same image size and structure of training and testing splits.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+A data point comprises an image and its label.
+
+### Data Fields
+
+- `image`: a 2d array of integers representing the 28x28 image.
+- `label`: an integer between 0 and 9 representing the classes with the following mapping:
+  | Label | Description |
+  | --- | --- |
+  | 0 | T-shirt/top |
+  | 1 | Trouser |
+  | 2 | Pullover |
+  | 3 | Dress |
+  | 4 | Coat |
+  | 5 | Sandal |
+  | 6 | Shirt |
+  | 7 | Sneaker |
+  | 8 | Bag |
+  | 9 | Ankle boot |
+
+### Data Splits
+
+The data is split into training and test set. The training set contains 60,000 images and the test set 10,000 images.
+
+## Dataset Creation
+
+### Curation Rationale
+
+The original MNIST dataset contains a lot of handwritten digits. Members of the AI/ML/Data Science community love this dataset and use it as a benchmark to validate their algorithms. In fact, MNIST is often the first dataset researchers try. "If it doesn't work on MNIST, it won't work at all", they said. "Well, if it does work on MNIST, it may still fail on others."
```

Comment:
This paragraph comes from kaggle right ? Or maybe another website ?
Would be nice to make it explicit that this comes from kaggle.

Reply:
This paragraph comes from the paper on arXiv. Infact, all information is here from the paper.

Reply:
Oh indeed you're right, nevermind

Reply:
I added "From the arXiv paper" everywhere.